pkg/util/liveconfig/hive.go

```diff
@@ -65,3 +66,17 @@ func (p *prod) HiveRestConfig(ctx context.Context, index int) (*rest.Config, err
 	p.cachedCredentials[index] = parsed
 	return rest.CopyConfig(parsed), nil
 }
+
+func (p *prod) InstallViaHive(ctx context.Context) (bool, error) {
+	// TODO: Replace with RP Live Service Config (KeyVault)
+	installViaHive := os.Getenv(HIVE_INSTALL_ENV_VARIABLE)
+	if installViaHive != "" {
+		return true, nil
+	}
+	return false, nil
+}
+
+func (p *prod) DefaultInstallerPullSpec(ctx context.Context) (string, error) {
+	// TODO: Replace with loading from M6's database always
+	return os.Getenv(HIVE_DEFAULT_INSTALLER_VARIABLE), nil
```

Comment:
No error will ever be returned here.

Reply:
The error returned in the interface is for when it is eventually fetched from keyvault.